cmd/get_versions.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	getVersionsName             = "get-versions"
+	getVersionsShortDescription = "Display info about supported Kubernetes versions"
+	getVersionsLongDescription  = "Display supported Kubernetes versions and upgrade versions"
+)
+
+func newGetVersionsCmd() *cobra.Command {
+	oc := orchestratorsCmd{}
+
+	command := &cobra.Command{
+		Use:   getVersionsName,
+		Short: getVersionsShortDescription,
+		Long:  getVersionsLongDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return oc.run(cmd, args)
+		},
+	}
+
+	f := command.Flags()
+	oc.orchestrator = "Kubernetes" // orchestrator is always Kubernetes
+	f.StringVar(&oc.version, "version", "", "Kubernetes version (optional)")
+	f.BoolVar(&oc.windows, "windows", false, "Kubernetes platform (optional)")
```

Comment:
I know this is leftover from the orchestrators command but I don't feel like this description is very accurate: maybe something like "Kubernetes cluster with Windows nodes".

I don't think my suggestion is much better but I couldn't come up with something good 😬 If `Kubernetes platform` makes sense to everyone feel free to disregard my comment.

Reply:
No, "Kubernetes platform" doesn't immediately make sense. Your suggestion is better--I'll use that or something similar.